extensions/nba-game-viewer/src/components/Player.tsx

```diff
@@ -0,0 +1,52 @@
+import type { Player } from "../types/roster.types";
+import { List, ActionPanel, Action, Icon } from "@raycast/api";
+import generatePlayerAccessories from "../utils/generatePlayerAccessories";
+
+type PropTypes = {
+  player: Player;
+  setIsShowingDetail: any;
+};
+
+const PlayerComponent = ({ player, setIsShowingDetail }: PropTypes) => {
+  return (
+    <List.Item
+      key={player.id}
+      title={player.fullName}
+      icon={player.headshot}
+      subtitle={player.position}
+      accessories={generatePlayerAccessories(player)}
+      detail={
+        <List.Item.Detail
+          markdown={`<img src="${player.headshot}" alt="drawing" width="250"/>`}
+          metadata={
+            <List.Item.Detail.Metadata>
+              <List.Item.Detail.Metadata.Label title="Position" text={player.position} />
```

Comment:
Maybe add the player name in the first row, I can't see the name
![CleanShot 2022-06-04 at 07 52 05@2x](https://user-images.githubusercontent.com/38033107/171986512-ade4c426-1707-4368-8522-d4d3102445a9.png)